src/main/java/seedu/address/logic/commands/DeleteNotesCommand.java

```diff
@@ -0,0 +1,93 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.util.ArrayList;
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Note;
+import seedu.address.model.person.Person;
+
+/**
+ * Deletes notes of an existing person in BizBook.
```

Comment:
```suggestion
 * Deletes notes of an existing person in the address book.
```
It would be better to avoid "BizBook" as is convention in the rest of the code. Especially if the name changes in the future.